exercises/ansible_rhel/1.1-setup/README.es.md

```diff
@@ -1,98 +1,143 @@
-* [Paso 2 - Trabajar los laboratorios](#Paso-2---Trabajar-los-laboratorios)
-* [Paso 3 - Laboratorios de desafío](#Paso-3---Laboratorios-de-desafío)
+- [Ejercicio del Taller - Verificar los Prerrequisitos](#ejercicio-del-taller---verificar-los-prerrequisitos)
+  - [Tabla de Contenidos](#tabla-de-contenidos)
+  - [Objetivo](#objetivo)
+  - [Guía](#guía)
+    - [Tu Entorno de Laboratorio](#tu-entorno-de-laboratorio)
+    - [Paso 1 - Acceder al Entorno](#paso-1---acceder-al-entorno)
+    - [Paso 2 - Usando la Terminal](#paso-2---usando-la-terminal)
+    - [Paso 3 - Examinar los Entornos de Ejecución](#paso-3---examinar-los-entornos-de-ejecución)
+    - [Paso 4 - Examinar la configuración de ansible-navigator](#paso-4---examinar-la-configuración-de-ansible-navigator)
+    - [Paso 5 - Labs de Desafío](#paso-5---labs-de-desafío)
 
-# Objetivos
+## Objetivo
 
-- Comprender la topología de laboratorio y cómo acceder al entorno.
-- Comprender cómo trabajar los ejercicios del taller
-- Comprender los laboratorios de desafío
+* Entender la Topología del Laboratorio: Familiarízate con el entorno de laboratorio y los métodos de acceso.
+* Dominar los Ejercicios del Taller: Adquiere competencia en la navegación y ejecución de las tareas del taller.
+* Abrazar los Labs de Desafío: Aprende a aplicar tus conocimientos en escenarios prácticos de desafío.
 
-# Guía
+## Guía
 
-## Su entorno de laboratorio
+La fase inicial de este taller se centra en las utilidades de línea de comandos de la Plataforma de Automatización Ansible, como:
 
-En este laboratorio se trabaja en un entorno de laboratorio preconfigurado. Tendrá acceso a los siguientes hosts:
 
-| Role                 | Inventory name |
-| ---------------------| ---------------|
-| Ansible Control Host | ansible        |
-| Managed Host 1       | node1          |
-| Managed Host 2       | node2          |
-| Managed Host 3       | node3          |
+- [ansible-navigator](https://github.com/ansible/ansible-navigator) - una Interfaz de Usuario basada en Texto (TUI) para ejecutar y desarrollar contenido de Ansible.
+- [ansible-core](https://docs.ansible.com/core.html) - el ejecutable base que proporciona el marco, lenguaje y funciones que sustentan la Plataforma de Automatización Ansible, incluyendo herramientas de línea de comandos como `ansible`, `ansible-playbook` y `ansible-doc`.
+- [Entornos de Ejecución](https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html) - Imágenes de contenedor pre-construidas con colecciones soportadas por Red Hat.
+- [ansible-builder](https://github.com/ansible/ansible-builder) - automatiza el proceso de construcción de Entornos de Ejecución. No es un enfoque principal en este taller.
 
-## Paso 1 - Acceso al entorno
+Si necesitas más información sobre los nuevos componentes de la Plataforma de Automatización Ansible, guarda esta página principal [https://red.ht/AAP-20](https://red.ht/AAP-20)
```

Comment:
[...] nuevos componentes de Ansible Automation Platform [...]

(no lo traduciría al ser nombre del producto)

La URL lleva a la KCS: https://access.redhat.com/articles/6145072 que parece llegar hasta AAP 2.1

Reply:
gonna leave this as is for now